src/assets/styles/theme.scss

```diff
@@ -48,9 +48,6 @@ $BCgovAGold2: #fff8ef;
 $BCgovAGold3: #ffe0bc;
 $BCgovAGold4: #ffd4a2;
 
-// Error Colors
-$BCgovInputError: #ff5252;
```

Comment:
This is a global (and hopefully common to our other projects) colour definition file. Are we replacing this colour with another? If not then please leave this definition.

Reply:
Yes it is been replaced with $app-red. Based on a comment in invisionapp

Reply:
Ah, ok!